### ISSUE TITLE
Use with Chart.js Error

### ISSUE THREAD

**Issue by tomherold** [3dcl/chartjs-plugin-responsive-downsample#7]
I am getting the following error:
chart.downsample.js:204 Uncaught TypeError: Cannot read property 'getTime' of undefined

Probably because I am using chart.min.js, however, I am including also moment.js before that.

**Comment by ckyycc** [comment]
Are you using "Number" datasets (with labels)? Currently only "Point" datasets are supported.

**Comment by tomherold** [comment]
I am using number datasets. Any chance to update your plugin to support number sets?

**Comment by ckyycc** [comment]
From my point of view, it's really hard to support number sets unless chart.js can do multiple labels. 
Otherwise, we will run into following situation:

Let's say we have below data (with 2 datasets and the labels):
```javascript
data: {
    labels: ["A", "B", "C", "D", "E", "F"],
    datasets: [
        { label: 'label1', data: [12, 19, 4, 5, 2, 3] },  
        { label: 'label2',  data: [11, 15, 1, 9, 3, 2] }
    ]
}
```
And, let's assume, after being downsampled,
dataset lable1 will become: data: [19, 5, 3],
dataset lable2 will become: data: [11, 1, 3].

Then, we do not know which labels need to use, ["B","D","F"] or ["A","C","E"] ? 

@MPursche correct me if I am wrong.

**Comment by MPursche** [comment]
@ckyycc Yes, only "Point" datasets are currently supported and I don't think there is a straight-forward way to add support for number sets. The implemented algorithms downsample data by aggregation or by leaving out data points. This only makes sense when there is a numerical or time-based x-axis.

**Comment by kmlbgn** [comment]
I have a time-based x-axis, any tips on a way to go around this problem?
